src/radical/pilot/agent/scheduler/archive/continuous_summit.py

```diff
@@ -557,29 +574,23 @@ def _find_resources(self, node, requested_cores, requested_gpus,
                 # We convert the number of usable cores into a number of procs so that
                 # we can find the minimum number of procs across lfs, cpus, gpus that can
                 # be allocated on this node
-                num_procs.append(usable_cores/core_chunk)
-            else:
-                alloc_cores = min(requested_cores, free_cores)
-                num_procs.append(alloc_cores / core_chunk)
+                num_procs['cores'] = usable_cores / core_chunk
 
         if requested_gpus:
             alloc_gpus = min(requested_gpus, free_gpus)
-            num_procs.append(alloc_gpus / gpu_chunk)
-
-        # Find min number of procs determined across lfs, cores, gpus
-        num_procs = min(num_procs)
+            num_procs['gpus'] = alloc_gpus / gpu_chunk
 
         # Find normalized lfs, cores and gpus
-        if requested_lfs:
-            alloc_lfs = num_procs * lfs_chunk
-        if requested_cores:
-            alloc_cores = num_procs * core_chunk
-        if requested_gpus:
-            alloc_gpus = num_procs * gpu_chunk
+        if requested_cores: alloc_cores = num_procs['cores'] * core_chunk
```

Comment:
I am a bit confused. Why do you divide it if you are going to multiply it at the end with `core_chuck` and render both operations unnecessary? Unless it is for rounding reasons, then I can see why it is needed